microceph/api/services.go

```diff
@@ -22,6 +22,7 @@ var servicesCmd = rest.Endpoint{
 	Path: "services",
 
 	Get: rest.EndpointAction{Handler: cmdServicesGet, ProxyTarget: true},
+	Put: rest.EndpointAction{Handler: cmdServicesPut, ProxyTarget: false},
```

Comment:
I'd suggest to use a POST request here, as with PUT there is an expectation that requests are idempotent
Also I'd lean towards introducing a separate endpoint e.g. /services/bootstrap for this to make explicit which operation is intended

Reply:
well, my only reservation about that would be that it diverges from the `services/{name}` that we follow for the services endpoint.

Reply:
The endpoint for the time being is changed to "ops/bootstrap", subject to change if the discussion with microcluster changes how we pull this up.